ods_ci/tasks/Resources/Provisioning/GPU/gpu_deploy.sh

```diff
@@ -1,17 +1,25 @@
 #!/bin/bash
 set -e
 
-# Make changes to gpu install file
+echo "Create and apply 'gpu_install.yaml' to install Nvidia GPU Operator"
 
 GPU_INSTALL_DIR="$(dirname "$0")"
 
-CHANNEL=$(oc get packagemanifest gpu-operator-certified -n openshift-marketplace -o jsonpath='{.status.defaultChannel}')
+CHANNEL="$(oc get packagemanifest gpu-operator-certified -n openshift-marketplace -o jsonpath='{.status.defaultChannel}')"
 
-CSVNAME=$(oc get packagemanifests/gpu-operator-certified -n openshift-marketplace -ojson | jq -r '.status.channels[] | select(.name == "'$CHANNEL'") | .currentCSV')
+CSVNAME="$(oc get packagemanifests/gpu-operator-certified -n openshift-marketplace -o json | jq -r '.status.channels[] | select(.name == "'$CHANNEL'") | .currentCSV')"
 
-sed -i -e "0,/v1.11/s//$CHANNEL/g" -e "s/gpu-operator-certified.v1.11.0/$CSVNAME/g"  ${GPU_INSTALL_DIR}/gpu_install.yaml
+sed -i -e "0,/v1.11/s//$CHANNEL/g" -e "s/gpu-operator-certified.v1.11.0/$CSVNAME/g"  "$GPU_INSTALL_DIR/gpu_install.yaml"
 
-oc apply -f ${GPU_INSTALL_DIR}/gpu_install.yaml
+oc apply -f "$GPU_INSTALL_DIR/gpu_install.yaml"
+
+echo "Wait for Nvidia GPU Operator Subscription, InstallPlan and Deployment to complete"
+
+oc wait --timeout=3m --for condition=CatalogSourcesUnhealthy=false subs nfd -n nvidia-gpu-operator
+
+oc wait installplan -n nvidia-gpu-operator --all --for condition=Installed --timeout=3m
+
+oc rollout status -n nvidia-gpu-operator deployment gpu-operator --watch --timeout=3m
```

Comment:
I don't think so rollout is necessary. Also could you add the check for nvidia operator as well

Reply:
I've updated the script to also wait for `gpu-operator-certified` subscription, and for the `nfd-controller-manager` deployment.
The rollout status is still needed, see the updated execution in my previous comment.